service/co/hotwax/configuration/CompanyConfigurationServices.xml

```diff
@@ -216,6 +216,11 @@
                 <service-call name="co.hotwax.configuration.UserProfileServices.get#OrganizationPartyIds" out-map="context"/>
                 <set field="organizationPartyId" from="context.partyId" type="String"/>
             </if>
+            <set field="fileName" from="sshKey.getName()"/>
+            <if condition="!fileName.endsWith('.pem')">
```

Comment:
Instead of relying only on .pem file extension, we should find another way to verify the ssh key file. 
Please refer SimpleSinger.readKey method for reference.